pinot-query-planner/src/main/java/org/apache/calcite/sql/fun/PinotOperatorTable.java

```diff
@@ -207,6 +222,49 @@ public final void initNoDuplicate() {
     }
   }
 
+  /**
+   * This registers transform functions defined in {@link org.apache.pinot.common.function.TransformFunctionType}
+   * which are multistage enabled.
+   */
+  private void initTransformFunctions() {
```

Comment:
see my change in https://github.com/apache/pinot/pull/11068 
this method can be greatly simplified

Reply:
ack